docs/guides/templates/profile.adoc

```diff
@@ -0,0 +1,11 @@
+<#macro ifProduct>
+ifeval::[{project_product} == true]
+<#nested>
+endif::[]
+</#macro>
+
+<#macro ifCommunity>
+ifeval::[{project_product} != true]
```

Comment:
I assume we can't use the same [approach](https://github.com/keycloak/keycloak/blob/61ca260d379d86e0a6862cab46fe9a0d7abb376d/docs/guides/templates/profile.adoc?plain=1#L8) as in `release/22.0` branch because in upstream we don't set the `project_product` and `project_community` properly?

Reply:
Exactly. The same approach as in `release/22.0` doesn't provide the functionality to use the `ifCommunity` in upstream. It means when you build a guide in upstream with the `ifCommunity` condition, it's hidden. This approach resolves it. This is mainly for the following product releases.

Reply:
Thanks for the confirmation. It's a bit odd that `project_community` is not apparently set properly in upstream. But this approach works for me too.

Reply:
As we're using AsciiDoctor in the upstream, we're facing a difference between prod and community documentation. Even if the `project_community` is specified as an AsciiDoctor boolean attribute, it behaves like a toggle (defined or non-defined)[1]. It means we're not able to check these conditions like `[{project_community} == true]`. We can only do that in product documentation, where these attributes are still considered as strings :))

Probably @ahus1 might have more information about it.

[1] https://docs.asciidoctor.org/asciidoc/latest/attributes/boolean-attributes/

Reply:
However, I've noticed we're using the Asciidoctor even for the `docs/documentation` and these attributes work properly. I'll try to investigate it a little bit.